src/js/form-model.js

```diff
@@ -86,6 +82,8 @@ FormModel.prototype = {
 
 /**
  * Initializes FormModel
+ *
+ * @return {Array<Error>} list of initialization errors
```

Comment:
This is an array of error messages (strings) not Error instances

Reply:
Fixed